pkg/app/api/service/apiservice/service.proto

```diff
@@ -92,3 +94,12 @@ message GetCommandRequest {
 message GetCommandResponse {
     pipe.model.Command command = 1;
 }
+
+message PushImageReferenceRequest {
+    string repo_name = 1 [(validate.rules).string.min_len = 1];
+    string tag = 2 [(validate.rules).string.min_len = 1];
```

Comment:
Although we should keep the tag in the model as a single string, at least this should be an array.

Reply:
Right